src/cmd/api/router/router.go

```diff
@@ -28,7 +28,7 @@ import (
 type Clients struct {
 	S3Client            clients.IS3Client
 	AmqpClient          clients.IAmqpClient
-	AmqpExchangerStatus clients.IAmqpExchanger
+	AmqpExchangerStatus clients.IAmqpClient
```

Comment:
Even better for the caller : 

```suggestion
	AmqpExchangerStatus clients.AmqpClient
```